mmdet/datasets/pipelines/loading.py

```diff
@@ -11,19 +11,30 @@
 @PIPELINES.register_module()
 class LoadImageFromFile(object):
 
-    def __init__(self, to_float32=False, color_type='color'):
+    def __init__(self,
+                 to_float32=False,
+                 color_type='color',
+                 file_client_args=dict(backend='disk')):
```

Comment:
Better add docstring and the allowed backends format here.

Reply:
By the way, `__repr__` method should also be modified to print out backend info.